src/HolographicCamera.Unity/Assets/HolographicCamera/Scripts/CalibrationDataProvider.cs

```diff
@@ -60,7 +60,7 @@ private async void SendCalibrationDataAsync()
                     message.Write("CalibrationData");
                     message.Write(contents.Length);
                     message.Write(contents);
-                    networkManager.Broadcast(memoryStream.ToArray());
+                    networkManager.Broadcast(memoryStream.GetBuffer(), 0, memoryStream.Position);
```

Comment:
>networkManager.Broadcast(memoryStream.GetBuffer(), 0, memoryStream.Position); [](start = 20, length = 77)

Can we use TryGetBuffer here? that will still work in .NET Scripting. The Azure Kinect changed all of the GetBuffer usages to TryGetBuffer elsewhere